### PR TITLE
added Norse Airlines page

### PR DESCRIPTION
**Pull request by AdityaBansal1402**
added norse airline page and also changed canadian-north's image because it was not working now it is

### CODE REVIEW

File: application/views/canadian-north.php [hunk 1]
```diff
@@ -3,7 +3,8 @@
 document.getElementsByTagName('meta')["keywords"].content = "";       
 document.getElementsByTagName('meta')["description"].content = "Find cheap flight at Tourcrusier. Get the best fairs With US - call us @ <?= TFN ?>";   
 </script>
-   <div id="header-wrapper" class="wrap-inpg py-4 py-md-5"  style="background-image: url('frontend/images/canadian-north.jpg'); background-position: bottom;background-no-repeat: no-repeat; background-size: cover;">
```
Comment (xxXWarMachineRoXxx) [inline application/views/canadian-north.php:6]: please add this path back and remove the other one

File: application/views/norse-airlines.php [hunk 2]
```diff
@@ -0,0 +1,236 @@
+<script>
+document.title = "Book Cheap Flight Ticket on Norse Airlines | Get Cheap Flight Deal Norse Airlines - Airfaremoss";       
+document.getElementsByTagName('meta')["keywords"].content = "";       
+document.getElementsByTagName('meta')["description"].content = "Find cheap flight at Tourcrusier. Get the best fairs With US - call us @ <?= TFN ?>";   
+</script>
+<!-- <div id="header-wrapper" class="wrap-inpg py-4 py-md-5"  style="background-image: url('frontend/images/Norse-Airlines.png'); background-position: bottom;background-no-repeat: no-repeat; background-size: cover;"> -->
+<div id="header-wrapper" class="wrap-inpg py-4 py-md-5 image " style="background-image: url('/airfaremoss.com/frontend/images/Norse-Airlines.png'); background-position: bottom;background-no-repeat: no-repeat; background-size: cover;">
```
Comment (xxXWarMachineRoXxx) [inline application/views/norse-airlines.php:7]: here too!

File: frontend/images/canadian-north.png
Comment (xxXWarMachineRoXxx) [inline frontend/images/canadian-north.png:1]: was the png not required ??

**Review by xxXWarMachineRoXxx — CHANGES_REQUESTED** [review]
please make the changes requested

**Comment by AdityaBansal1402** [comment]
it won't work on loacalhost but will work on the website?

File: frontend/images/canadian-north.png
Comment (AdityaBansal1402) [reply frontend/images/canadian-north.png:1]: actually the extension was for jpg could have just changed the extension but thought changing the type of the image would show it ofc it didn't but there is no change just a png changed to jpg

**Comment by AdityaBansal1402** [comment]
did the required changes

File: application/views/norse-airlines.php
[the remark this answers is not in the document]
Comment (xxXWarMachineRoXxx) [reply application/views/norse-airlines.php:1]: 👍🏼

Comment (xxXWarMachineRoXxx) [inline application/views/norse-airlines.php:1]: 👍🏼

**Review by xxXWarMachineRoXxx — COMMENTED** [review]
what about the images?

**Review by xxXWarMachineRoXxx — APPROVED** [review]
Okay then , Seems correct.

🟢 Commit is go , I repeat Commit is go !!

**Pull request merged**